### PR TITLE
Remove vendor dir

### PR DESCRIPTION
**Pull request by mrKisaoLamb**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	github.com/hashicorp/golang-lru v0.5.4
+	github.com/imdario/mergo v0.3.8
+	github.com/joho/godotenv v1.3.0
+	github.com/json-iterator/go v1.1.9
+	github.com/mailru/easyjson v0.7.0
+	github.com/markbates/inflect v1.0.4
+	github.com/matttproud/golang_protobuf_extensions v1.0.1
+	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
+	github.com/modern-go/reflect2 v1.0.1
+	github.com/operator-framework/operator-sdk v0.8.3-0.20190722210327-daf62d44e47e
```
Comment (stuggi) [inline go.mod:44]: does this PR needs to be updated after you changed the other one?

Comment (mrKisaoLamb) [reply go.mod:44]: Yeah I will rebase when previous one will be merged

**Comment by dprince** [comment]
lgtm

**Comment by dprince** [comment]
@mrKisaoLamb just merged your other request. rebase this one and then we can merge it

**Comment by mrKisaoLamb** [comment]
@dprince @stuggi  rebase done

**Pull request merged**